Source/Core/Core/HW/WiimoteEmu/WiimoteEmu.cpp

```diff
@@ -770,7 +770,7 @@ void Wiimote::ControlChannel(const u16 _channelID, const void* _pData, u32 _Size
 	if (99 == _channelID)
 	{
 		// Wiimote disconnected
-		//PanicAlert( "Wiimote Disconnected" );
+		//PanicAlert("Wiimote Disconnected");
```

Comment:
Style-fixing commented-out code, yay. Can't we just drop that line, or does it serve any particular purpose?